packages/react-ui-core/src/Counter/__tests__/Counter-test.js

```diff
@@ -101,6 +102,23 @@ describe('Counter', () => {
       expect(wrapper.state('count')).toEqual(3)
     })
 
+    it('does not allow clicks to propogate from outer div', () => {
+      const onClick = jest.fn()
+      const wrapper = mount(
+        <Counter
+          min={1}
+          count={1}
+          max={3}
+          theme={theme}
+          className="wrappingDiv"
+          onClick={onClick}
+        />
+      )
+      const outerDiv = wrapper.find('div.wrappingDiv')
```

Comment:
I don't think you need to `find` anything.  By default the wrapper should be the root node which is what you want to simulate.  I think you can just do the following instead:
```
  wrapper.simulate('click')
  expect(wrapper.prop('onClick')).not.toBeCalled()
})
```